Engine/Graphics/Direct3D/Render.cpp

```diff
@@ -2021,7 +2038,7 @@ void Render::DrawIndoorPolygon(unsigned int uNumVertices, BLVFace *pFace,
     TextureD3D *texture = (TextureD3D *)pFace->GetTexture();
 
     if (lightmap_builder->StationaryLightsCount) {
-        sCorrectedColor =  0xFFFFFFFF/*-1*/;
+        sCorrectedColor =  0xFFFFFF/*-1*/;
```

Comment:
Can you comment on the semantics here?

Reply:
Change made when trying to track down a bug. Will revert.